cmake/sca/eclair/sca.cmake

```diff
@@ -21,75 +21,117 @@ set(CXX_ALIASES "${CMAKE_CXX_COMPILER}")
 set(AS_ALIASES "${CMAKE_AS}")
 set(LD_ALIASES "${CMAKE_LINKER}")
 set(AR_ALIASES "${CMAKE_ASM_COMPILER_AR} ${CMAKE_C_COMPILER_AR} ${CMAKE_CXX_COMPILER_AR}")
-if(NOT ECLAIR_RULES_SET)
-  set(ECLAIR_RULES_SET first_analysis)
+
+set(ECLAIR_ENV_ADDITIONAL_OPTIONS "")
+set(ECLAIR_REPORT_ADDITIONAL_OPTIONS "")
+
+# Default value
+set(ECLAIR_RULES_SET first_analysis)
+
+if (ECLAIR_CONFIG)
```

Comment:
General question is this working with the kconfig files? Did you test it?

Reply:
I found a typo on a specific configuration (fixed in my last commit), but I don't think you are referring to it.

I think your question arises from the fact that ECLAIR_CONFIG is undefined. 
The reason is that in my mind, the location of the .config file should be provided by the west invocation:
`west build -b mimxrt1064_evk samples/basic/blinky -- -DZEPHYR_SCA_VARIANT=eclair -DECLAIR_CONFIG=$(pwd)/cmake/sca/eclair/.config`

I forgot to tell you, my bad. Sorry.

Reply:
Of course, if you think there is a default place where the .config file should be placed we can 
set ECLAIR_CONFIG to this place, but this, IMO, depends on how we want to integrate the Kconfig.

Maybe the community wants to integrate it into a more general one, or maybe not.
I don't know which is the standard way to proceed.

Reply:
Sorry for the late response and thanks for adding this. About where to add the config and all of the configuration is a good point.
As far as I understood tejlmand is that the kconfig is right now not allowed for SCA tools in the project.
But with your changes we can might propose how it can be used but i need to think about how we can place this in the PR or maybe we need to split this part out.